### DISCUSSION TITLE
【提案】数组去重方法

### DISCUSSION THREAD

**Discussion by TechQuery** [JSCIG/es-discuss#13] [Stage 1]
`[...new Set(array)]` 只能处理**原始类型值**，对象等复杂**引用类型**中的数据因为引用的不同而不适用，就需要一个**按一定条件去重**的方法 —— `Array.prototype.unique()` ，来应付诸如数据库查出的数据（唯一 ID）或 UI 中用户操作生成的一些冗杂数据。

- repository: https://github.com/TechQuery/array-unique-proposal
- champion: @Jack-Works

**Comment by hax** [comment]
恭喜进入Stage 1！
